includes/settings/js/src/components/Taxonomies.jsx

```diff
@@ -0,0 +1,505 @@
+								className={
+									errors.plural ? "field has-error" : "field"
+								}
+							>
+								<label htmlFor="plural">
+									{__("Plural Name", "atlas-content-modeler")}
+								</label>
+								<br />
+								<p className="help">
+									{__(
+										'Plural display name for your taxonomy, e.g. "Ingredients".',
+										"atlas-content-modeler"
+									)}
+								</p>
+								<input
+									id="plural"
+									name="plural"
+									placeholder="Ingredients"
+									className="w-100"
+									ref={register({
+										required: true,
+										maxLength: 50,
+									})}
+									onChange={(event) => {
+										setPluralCount(
+											event.target.value.length
+										);
+									}}
+								/>
+								<p className="field-messages">
+									{errors.plural &&
+										errors.plural.type === "required" && (
+											<span className="error">
+												<Icon type="error" />
+												<span role="alert">
+													{__(
+														"This field is required",
+														"atlas-content-modeler"
+													)}
+												</span>
+											</span>
+										)}
+									{errors.plural &&
+										errors.plural.type === "maxLength" && (
+											<span className="error">
+												<Icon type="error" />
+												<span role="alert">
+													{__(
+														"Exceeds max length.",
+														"atlas-content-modeler"
+													)}
+												</span>
+											</span>
+										)}
+									<span>&nbsp;</span>
+									<span className="count">
+										{pluralCount}/50
+									</span>
+								</p>
+							</div>
+
+							{/* API Identifier / Slug */}
+							<div
+								className={
+									errors.slug ? "field has-error" : "field"
+								}
+							>
+								<label htmlFor="slug">
+									{__(
+										"API Identifier",
+										"atlas-content-modeler"
+									)}
+								</label>
+								<br />
+								<p className="help">
+									{__(
+										"Auto-generated from the singular name and used for API requests.",
+										"atlas-content-modeler"
+									)}
+								</p>
+								<input
+									id="slug"
+									name="slug"
+									className="w-100"
+									ref={register({
+										required: true,
+										maxLength: 20,
+									})}
+									{...apiIdFieldAttributes}
+								/>
+								<p className="field-messages">
+									{errors.slug &&
+										errors.slug.type === "required" && (
+											<span className="error">
+												<Icon type="error" />
+												<span role="alert">
+													{__(
+														"This field is required",
+														"atlas-content-modeler"
+													)}
+												</span>
+											</span>
+										)}
+									{errors.slug &&
+										errors.slug.type === "maxLength" && (
+											<span className="error">
+												<Icon type="error" />
+												<span role="alert">
+													{__(
+														"Exceeds max length of 20.",
```

Comment:
```suggestion
														"Exceeds max length of 32",
```

to match above suggestion